onchain/example-program-on-bonsol/src/lib.rs

```diff
@@ -0,0 +1,155 @@
+            )
+            .map_err(|_| ProgramError::InvalidInstructionData)?;
+            invoke_signed(&ix, accounts, &[&[execution_id.as_bytes(), &[bump]]])?;
+            let mut data = requester.try_borrow_mut_data()?;
+            data.copy_from_slice(&execution_account.key.to_bytes());
+            Ok(())
+        }
+        1 => {
+            let requester = &accounts[1];
+            let requester_data = requester.try_borrow_data()?;
+            let execution_account = Pubkey::try_from(&requester_data[0..32])
+                .map_err(|_| ProgramError::InvalidInstructionData)?;
+            let callback_output: BonsolCallback =
+                handle_callback(SIMPLE_IMAGE_ID, &execution_account, accounts, data)?;
+            if sol_memcmp(accounts[2].key.as_ref(), EA1.as_ref(), 32) != 0 {
+                return Err(ProgramError::InvalidInstructionData.into());
+            }
+            if sol_memcmp(accounts[3].key.as_ref(), EA2.as_ref(), 32) != 0 {
+                return Err(ProgramError::InvalidInstructionData.into());
+            }
+            if sol_memcmp(accounts[4].key.as_ref(), EA3.as_ref(), 32) != 0 {
+                return Err(ProgramError::InvalidInstructionData.into());
+            }
+            assert!(accounts[2].is_writable, "Writable account not found");
+            if callback_output.committed_outputs.len() == 1
+                && callback_output.committed_outputs[0] == 1
+            {
+                msg!("Correct Json Attestation");
+            }
+            Ok(())
+        }
+        //only callback test
+        2 => {
+            let callback_output: BonsolCallback =
+                handle_callback(SIMPLE_IMAGE_ID, &accounts[0].key, accounts, data)?;
+            if sol_memcmp(accounts[1].key.as_ref(), EA1.as_ref(), 32) != 0 {
+                return Err(ProgramError::InvalidInstructionData.into());
+            }
+            if sol_memcmp(accounts[2].key.as_ref(), EA2.as_ref(), 32) != 0 {
+                return Err(ProgramError::InvalidInstructionData.into());
+            }
+            if sol_memcmp(accounts[3].key.as_ref(), EA3.as_ref(), 32) != 0 {
+                return Err(ProgramError::InvalidInstructionData.into());
+            }
+            assert!(accounts[1].is_writable, "Writable account not found");
+            if callback_output.committed_outputs.len() == 1
+                && callback_output.committed_outputs[0] == 1
+            {
+                msg!("Correct Json Attestation");
+            }
+            Ok(())
+        }
+
+        _ => return Err(ProgramError::InvalidInstructionData.into()),
+    }
+}
+
+pub fn create_program_account<'a>(
+    account: &'a AccountInfo<'a>,
+    seeds: &[&[u8]],
+    space: u64,
+    payer: &'a AccountInfo<'a>,
+    system: &'a AccountInfo<'a>,
+    additional_lamports: Option<u64>,
+) -> Result<(), ProgramError> {
+    let lamports =
+        Rent::default().minimum_balance(space as usize) + additional_lamports.unwrap_or(0);
```

Comment:
did you mean Rent::get()?   [https://docs.rs/solana-program/latest/solana_program/sysvar/trait.Sysvar.html#method.get](https://docs.rs/solana-program/latest/solana_program/sysvar/trait.Sysvar.html#method.get)